### PR TITLE
Prepare Release 1.26.0

### PR DESCRIPTION
**Pull request by cmslewis**
:fireworks: **Highlights**: Better `Table` column reordering, inline `RadioGroup` options

:book: **Latest docs**: [blueprintjs.com/docs](http://blueprintjs.com/docs)

### 💎  New features
- #1475 `RadioGroup` now supports the `inline` prop to render radio options inline. (🎩 @cathyxz)
image

### 🐛  Bug fixes
- #1303 `Hotkeys` dialog now shows <kbd>Cmd</kbd> or <kbd>Ctrl</kbd> appropriately in hotkey combos, depending on the client platform.
- #1476 `Tabs` of different sizes (regular or `pt-large`) can now be easily nested.
    - ⚠️  If you are nesting two more levels of `pt-large` `Tabs` in your application, you may need to add `pt-large` to each set of tabs as of this release.
- #1482 `NonIdealState` now properly line-wraps long text in Internet Explorer. (🎩 @AlexLandau)
image

### 🐛  Bug fixes
- #1480 `DateRangeInput` now uses provided `locale` properly when formatting dates. (🎩 @AlexLandau)

### 💎  New features
- #1462 `Table` column reordering is now much improved!
    - Reorderable columns now _always_ have a reorder handle, even if the interaction bar is disabled.
    - Columns can now be reordered even when selection is disabled.
    - Reordering disjoint selections is now much more intuitive due to improved visual feedback.
reordering

### 💎  New features
- #1465 `MultiSelect` tags can now contain arbitrary React elements. (🎩 @sushain97)

### 🐛  Bug fixes
- #1485 `MultiSelect` and `TagInput` styles are now more consistent with regular `pt-input`s. (🎩 @sushain97)

### CODE REVIEW

File: packages/datetime/package.json [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
-  "version": "1.20.0",
+  "version": "1.21.0",
   "description": "Components for interacting with dates and times",
   "main": "dist/index.js",
   "typings": "dist/index.d.ts",
   "style": "dist/blueprint-datetime.css",
   "unpkg": "dist/datetime.bundle.js",
   "dependencies": {
-    "@blueprintjs/core": "^1.3.0",
+    "@blueprintjs/core": "^1.26.0",
```
Comment (giladgray) [inline packages/datetime/package.json]: revert

File: packages/docs/package.json [hunk 2]
```diff
@@ -7,7 +7,7 @@
   "style": "dist/docs.css",
   "unpkg": "dist/docs.bundle.js",
   "dependencies": {
-    "@blueprintjs/core": "^1.25.1",
+    "@blueprintjs/core": "^1.26.0",
```
Comment (giladgray) [inline packages/docs/package.json]: revert

File: packages/labs/package.json [hunk 3]
```diff
@@ -1,13 +1,13 @@
...
-  "version": "0.6.0",
+  "version": "0.7.0",
   "description": "Incubator for unstable and in-development Blueprint components",
   "main": "dist/index.js",
   "typings": "dist/index.d.ts",
   "style": "dist/blueprint-labs.css",
   "unpkg": "dist/labs.bundle.js",
   "dependencies": {
-    "@blueprintjs/core": "^1.25.0",
+    "@blueprintjs/core": "^1.26.0",
```
Comment (giladgray) [inline packages/labs/package.json]: revert (i think?)

File: packages/site-docs/package.json [hunk 4]
```diff
@@ -4,11 +4,11 @@
   "description": "Blueprint Docs",
```
Comment (giladgray) [inline packages/site-docs/package.json]: bump version on line above to match __core__

File: packages/site-landing/package.json [hunk 5]
```diff
@@ -22,7 +22,7 @@
     "watch": "onchange 'src/**' -- npm-run-all build:copy build:webpack"
   },
   "devDependencies": {
-    "@blueprintjs/core": "^1.25.1",
+    "@blueprintjs/core": "^1.26.0",
```
Comment (giladgray) [inline packages/site-landing/package.json]: revert

File: packages/table/package.json [hunk 6]
```diff
@@ -20,7 +20,7 @@
     "watch": "onchange 'src/**' 'preview/*.ts*' -- npm-run-all build:gulp build:preview"
   },
   "dependencies": {
-    "@blueprintjs/core": "^1.23.0",
+    "@blueprintjs/core": "^1.26.0",
```
Comment (giladgray) [inline packages/table/package.json]: revert

File: packages/docs/package.json (hunk 2 above)
Comment (cmslewis) [inline packages/docs/package.json]: Done

File: packages/datetime/package.json (hunk 1 above)
Comment (cmslewis) [inline packages/datetime/package.json]: Done

File: packages/site-docs/package.json (hunk 4 above)
Comment (cmslewis) [inline packages/site-docs/package.json]: Done

File: packages/table/package.json (hunk 6 above)
Comment (cmslewis) [inline packages/table/package.json]: Done

File: packages/site-landing/package.json (hunk 5 above)
Comment (cmslewis) [inline packages/site-landing/package.json]: Done

File: packages/labs/package.json (hunk 3 above)
Comment (cmslewis) [inline packages/labs/package.json]: Nope, I think we need this. See: https://github.com/palantir/blueprint/pull/1485/files

**Comment by tgreenwatts** [comment]
Language looks good to me 👍

**Comment by AlexLandau** [comment]
"NonIdealState now properly line-wraps long text." -- Should probably specify "in Internet Explorer"?

**Comment by llorca** [comment]
That's... not great (unrelated to this release):
image

File: packages/labs/package.json (hunk 3 above)
Comment (giladgray) [inline packages/labs/package.json]: ✅ correct

**Comment by giladgray** [comment]
i think the `Tabs2 large` note is really a bug fix

**Comment by giladgray** [comment]
in fact, i'd argue both of those points are bug fies (Tabs2 and Hotkeys)

**Comment by cmslewis** [comment]
@giladgray updated 👍

**Pull request merged**